src/utils/events.ts

```diff
@@ -96,6 +96,9 @@ export const getEventData = (
   const noteTitle      = retrieveEventValue(
     eventObject, 'title', file.name.replace( '.md', '' ), frontMatterKeys?.titleKey
   )
+  const noteBody       = retrieveEventValue( 
+    eventObject, 'description', isHTMLElementType(eventObject) ? event.innerText : ''
```

Comment:
`event.innerText` needs to be `eventObject.innerText` in this context.

Also, the defaultValue param is a little messy, can you move it to it's own variable (perhaps above where `color` is declared?) for readability?

Reply:
Yes, I'll have to do a proper clone but after I get a working dev environment I can take care of that. Previously I was just tweaking things in the raw plugin js file and then I forked the repo and was manually editing files in the Github's web based texy editor to reflect the changes I made in my local js file. Admittedly sloppy